playbooks/awx-push-new-schema.yaml

```diff
@@ -0,0 +1,16 @@
+---
+- hosts: static
+  tasks:
+    - name: Generate new schema
+      command: make docker-compose-genschema
+
+    - name: Upload new schema
+      aws_s3:
+        bucket: awx-public-ci-files
+        object: schema.json
+        src: schema.json
+        mode: put
```

Comment:
need to add `permission: public-read` or we won't be able to `curl` this/won't be readable by other people

Reply:
https://docs.ansible.com/ansible/latest/modules/aws_s3_module.html